src/main/java/com/archisacadeny/instructor/InstructorRepository.java

```diff
@@ -47,6 +47,19 @@ public Instructor save (Instructor instructor){
         return instructor;
     }
 
+    public Instructor createInstructor(long id, String instructorNumber, String fullName, String email, String password, List<Course> taughtCourses){ //buradaki List<Course> kısmındaki course'ta hata alıyorum nden
+
+        Instructor instructor = new Instructor();
+        instructor.setId(id);
+        instructor.setNumber(instructorNumber);
+        instructor.setFullName(fullName);
+        instructor.setEmail(email);
+        instructor.setPassword(password);
+        instructor.setCourses(taughtCourses);
```

Comment:
course will be assigned to the instructor. you need to remove it from here has well from the method paramters